lib/cb/clients/resume_insights.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+require_relative 'base'
+module Cb
+  module Clients
+    class ResumeInsights < Base
```

Comment:
I wish this would have stuck with either Resume Insights or Keyword Insights - not a mix of both. The route is *keyword* but you name a lot of things in the API gem here *resume*.